frontend/src/Pages/CategoryListPage.tsx

```diff
@@ -48,24 +49,27 @@ const CategoryListPage = () => {
                   {category.description}
                 </td>
                 <td className="border-2 border-black whitespace-nowrap px-[5px] text-center">
+                  <Link to={{pathname: `/admin/category/${category.id}/addword`}}>
                   <button
                     type="button"
                     className="px-[5px] underline decoration-sky-500 hover:bg-sky-500"
                   >
                     Add Words
-                  </button>
+                  </button> </Link>
+                  <Link to={{pathname: `/admin/category/${category.id}/edit`}}>
                   <button
                     type="button"
                     className="px-[5px] underline decoration-yellow-500 hover:bg-yellow-700"
                   >
                     Edit
-                  </button>
+                  </button> </Link>
+                  <Link to={{pathname: `/admin/category/${category.id}/delete`}}>
                   <button
                     type="button"
                     className="px-[5px] underline decoration-red-500 hover:bg-red-500"
                   >
                     Delete
-                  </button>
+                  </button> </Link>
```

Comment:
format this properly. 

https://www.digitalocean.com/community/tutorials/how-to-format-code-with-prettier-in-visual-studio-code